setup.py

```diff
@@ -40,21 +40,20 @@
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
     ],
     packages=['nbflow'],
     scripts=['scripts/nbflow'],
     package_data={
         'nbflow': extension_files
-    }
+    },
+    install_requires=[
```

Comment:
If possible I would like to keep the way this code was before, since it means the requirements only need to be maintained in one place (in `requirements.txt`). Was there a reason this was changed here?

Reply:
First there was an error because the requirements.txt wasn't include in the distribution build, that was fix adding a MANIFEST.in in the new PR #12